src/TR_alg.jl

```diff
@@ -133,6 +133,7 @@ function TR(
 
   λmax = opnorm(Bk)
   νInv = (1 + θ) * λmax
+  ν = 1 / (νInv + 1 / (Δk * α))
```

Comment:
Shouldn’t this be 1 / (λmax + 1 / (Δk * α)) ?

Reply:
This was already done like this before this PR, I put it here so that $$\nu$$ is available outside the while loop.

Reply:
But I mean: why do we need the `θ` here?

Reply:
The file input_struct.jl says: "step length factor in relation to Hessian norm". It is small by default ($10^{-3}$) so I would guess that it is some kind of safety if the Hessian norm is underevaluated, but I can't be sure, I did not implement this.